src/app/Page/main-view/main-view.component.ts

```diff
@@ -30,22 +45,29 @@ export class MainViewComponent implements OnInit {
       .subscribe({
         next: (employees) => {
 
-          this.employees = employees  // *taking the value of responce and populating it into a variable.
+          // Dean you will want to split these into the collections defined above based on the value of status
+          this.employees[0].employees = employees  // *taking the value of responce and populating it into a variable.
```

Comment:
as per the comment, these need to be split on status, this just pushes everything into the first column if you get a response, 

you should be a bit more defensive, what if the call to the service fails,

I personally am not a bit fan of rxjs observables espechally when they will only yield one value,